src/orders/dto/create-order.dto.ts

```diff
@@ -0,0 +1,63 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { ArrayNotEmpty, IsArray, IsInstance, IsInt, IsString, Min, MinLength, ValidateNested } from "class-validator";
+import { CustomerDto } from "src/customers/dto/customer.dto";
+import { LocationDto } from "src/products/dto/location.dto";
+import { ProductDto } from "src/products/dto/product.dto";
+
+export class OrderItem{
```

Comment:
**issue:** please have separate .ts files for this

Reply:
Solved: Renamed it to OrderItemDto and moved it to /src/orders/dto/order-item-dto.ts